test/rpc/conftest.py

```diff
@@ -65,7 +65,6 @@ def fixture_invoke_info() -> dict:
     Make an invoke transaction on devnet and return invoke info dict
     """
     invoke_tx = json.loads(INVOKE_CONTENT)
-    invoke_tx["calldata"] = ["0"]
```

Comment:
yes one of the test will break

```
invoke_info = {'address': '0x0491c87978cb8d8a82c2a96dddd1888eb5ef8aa7209d5da86d2b703c3572032a', 'calldata': ['0'], 'code': 'TRANSACTION_RECEIVED', 'contract_address': '0x0491c87978cb8d8a82c2a96dddd1888eb5ef8aa7209d5da86d2b703c3572032a', ...}

    @pytest.mark.usefixtures("run_devnet_in_background", "deploy_info")
    def test_get_transaction_by_hash_invoke(invoke_info):
        """
        Get transaction by hash
        """
>       block = get_block_with_transaction(invoke_info["transaction_hash"])

test/rpc/test_rpc_transactions.py:69: 
```

because this line overrides the `calldata` parameters defined in the json. to be honest, i don't really know why the overrides happen in the `fixture` instead changing directly in the `test/invoke_rpc.json`. are there any contexts that i missed?

Reply:
Do you maybe know why `calldata` is overriden in the fixture @drknzz ?

Reply:
I have no idea why it's there.
Most likely a bug, as I do not see why it would be overwritten here (since calldata is defined in the file as
```
    "calldata": [
        "10"
    ],
```
)

Reply:
ok, thanks for taking a look